nilearn/plotting/tests/test_html_stat_map.py

```diff
@@ -332,6 +333,7 @@ def test_json_view_to_html(affine_eye, black_bg, cbar):
         annotate=False,
         title="test",
         colorbar=True,
+        radiological=False,
```

Comment:
let's parametrize this like is being done for cbar and black bg, because otherwise not all the new code is actually tested